e2e/tests/patient/patient-crud.e2e-spec.ts

```diff
@@ -0,0 +1,178 @@
+import { LoginPage } from '../../page-objects/login/login.po';
+import { MainPage } from '../../page-objects/main/main.po';
+import { PatientMaintenancePage } from '../../page-objects/patient/patient-maintenance.po';
+import { PatientDetailPage } from '../../page-objects/patient/patient-detail/patient-dialog.po';
+import { TestUtil } from '../../utilities/test-util';
+import { FormService, FormData, ButtonState } from '../../services/form.service';
+
+
+declare const allure: any;
+
+describe('TC0001_PatientManagement_e2e', () => {
+
+    const loginPage = new LoginPage();
+    const mainPage = new MainPage();
+    const patientMaintenancePage = new PatientMaintenancePage();
+    const patientDetailPage = new PatientDetailPage();
+
+    beforeAll(() => {
+        NavigationService.navigateToHomePage(loginPage);
+        NavigationService.login(loginPage, false);
+        NavigationService.navigateToPatientMaintenancePage(mainPage);
+    });
+
+    beforeEach(() => {
+        TestUtil.init('TC0001_PatientManagement_e2e', 'Purpose: This TC is intended to verify the CRUD of a Patient',
+            loginPage.appVersion, 'userName');
+    });
+
+    /* Base data to fill/check in form Create/Update patients*/
+    function getFormData(i?: number): FormData[] {
+        const basePatientValues = ['Surname', 'Name', 'email@werfen.com', 'Plaza de Europa, 21-23', 'Barcelona', '08908', '41.356439, 2.127791'];
+
+        const empty = (i === undefined);
+        const formData: FormData[] = [{
+            field: patientDetailPage.getSurnameInput().getElement(),
+            name:  'Surname',
+            value: empty ? '' : 'Try #' + i + ': ' + basePatientValues[0]
+        }, {
+            field: patientDetailPage.getNameInput().getElement(),
+            name:  'Name',
+            value: empty ? '' : 'Try #' + i + ': ' + basePatientValues[1]
+        }, {
+            field: patientDetailPage.getEmailInput().getElement(),
+            name:  'Email',
+            value: empty ? '' : 'try_' + i + '_' + basePatientValues[2]
+        }, {
+            field: patientDetailPage.getAddressStreetInput().getElement(),
+            name:  'Address -> Street',
+            value: empty ? '' : 'Try #' + i + ': ' + basePatientValues[3]
+        }, {
+            field: patientDetailPage.getAddressCityInput().getElement(),
+            name:  'Address -> City',
+            value: empty ? '' : 'Try #' + i + ': ' + basePatientValues[4]
+        }, {
+            field: patientDetailPage.getAddressZipInput().getElement(),
+            name:  'Address -> Zip',
+            value: empty ? '' : 'Try #' + i + ': ' + basePatientValues[5]
+        }, {
+            field: patientDetailPage.getAddressCoordinatesInput().getElement(),
+            name:  'Address -> Coordinates',
+            value: empty ? '' : 'Try #' + i + ': ' + basePatientValues[6]
+        }];
+        return (formData);
+    }
+
+    it('Open Patient Creation Dialog', () => {
```

Comment:
Each action must have at least an expected result. Otherwise, we need to have all the actions inside the same row in the TC with the final expected result.

See the report in Allure 
![image](https://user-images.githubusercontent.com/1970034/69874816-3b85d680-12bc-11ea-9568-95ce8d63cc98.png)